common/shlibs

```diff
@@ -3463,3 +3463,4 @@ libgrpc_unsecure.so.7 grpc-1.19.1_1
 libgrpcpp_channelz.so.1 grpc-1.19.1_1
 libircclient.so.1 libircclient-1.10_1
 libFAudio.so.0 FAudio-19.05_1
+libqaccessibilityclient-qt5.so.0 libqaccessibilityclient-devel-0.4.0_1
```

Comment:
this entry should point to the main package, not `-devel`

Reply:
fixed